javageci-core/src/main/java/javax0/geci/accessor/Accessor.java

```diff
@@ -1,16 +1,16 @@
 package javax0.geci.accessor;
 
-import javax0.geci.api.Segment;
-
 import java.lang.reflect.Field;
+import javax0.geci.api.Segment;
+import javax0.geci.api.Source;
```

Comment:
I usually refuse to accept pull requests that are low quality. Not having unused imports in the source file is a quality requirement. Simple to check, easy to measure, easy to fix.